### DISCUSSION TITLE
Resizable Monaco Editor widget based dialog

### DISCUSSION THREAD

**Discussion by kapishsingla** [eclipse-theia/theia#13704] [Q&A] [answered]
Hi,
We are using the Monaco editor embedded within widget dialog box. Now we are aiming to increase dimension of Monaco editor by dragging the boundary of Dialog Box.

![Screenshot 2024-04-05 121053](https://github.com/eclipse-theia/theia/assets/132569352/9b54be58-8d1e-4855-95af-fe39ff549974)

I am currently using theia v1.43.1.

Request you to help with an approach for same?

**Comment by msujew** [comment, marked as answer]
Hey @kapishsingla,

I haven't tried any of that yet, but the size of dialogs are purely limited using CSS. For your use case, see [here](https://www.w3schools.com/cssref/css3_pr_resize.php).
